src/applications/disability-benefits/all-claims/content/ratedDisabilities.jsx

```diff
@@ -8,12 +8,23 @@ import { NULL_CONDITION_STRING } from '../constants';
  * @property {String} diagnosticCode
  * @property {String} name
  * @property {String} ratingPercentage
+ * @property {String} maximumRatingPercentage
  * @param {Disability} disability
  */
-export const disabilityOption = ({ name, ratingPercentage, className }) => {
+export const disabilityOption = ({
+  name,
+  ratingPercentage,
+  className,
+  maximumRatingPercentage,
+}) => {
   // May need to throw an error to Sentry if any of these doesn't exist
   // A valid rated disability *can* have a rating percentage of 0%
   const showRatingPercentage = Number.isInteger(ratingPercentage);
+  const showMaxRatingMessage =
+    Number.isInteger(maximumRatingPercentage) &&
+    maximumRatingPercentage === ratingPercentage &&
```

Comment:
could there ever be an exception case where a veteran's rating is above the max? if so, do we want to check >=? If not, ignore.

Reply:
It is actually possible! But the circumstances that lead to a rating exceeding the maximum are rare and unusual, and if the Veteran is in that situation it's debatable whether this education is even meaningful for them. So for this pilot program we're going to just tackle the `==` case and measure its impact first.